mock/cpp_mock_scenarios/src/metrics/reaction_time.cpp

```diff
@@ -30,10 +29,7 @@ class ReactionTimeScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit ReactionTimeScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "reaction_time",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.